apps/condo/domains/miniapp/schema/B2BAppAccessRight.test.js

```diff
@@ -186,5 +191,64 @@ describe('B2BAppAccessRight', () => {
                 })
             }, NON_SERVICE_USER_ERROR)
         })
+
+        test('One service user can be linked for different B2BApps', async () => {
+            const admin = await makeLoggedInAdminClient()
+            const [serviceUser] = await registerNewServiceUserByTestClient(admin)
+
+            const [app] = await createTestB2BApp(admin)
+            const [app2] = await createTestB2BApp(admin)
+
+            await createTestB2BAppAccessRight(admin, serviceUser, app)
+            await createTestB2BAppAccessRight(admin, serviceUser, app2)
```

Comment:
Maybe expect something? (at least that it is defined) :)

Reply:
This is enough to catch the error. 
But I'll add it to make it more obvious)